desktop.css

```diff
@@ -0,0 +1,928 @@
+    /* Toolbar section */
+    #toolbar {
+        background: var(--TOOLBAR-BACKGROUND-COLOR);
+        height: 72px;
+        width: 100%;
+        position: fixed;
+    }
+    #toolbar-menu-button{
+        display: none;
+    }
+    #toolbar-logo-placeholder {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        width: 116px;
+        height: 40px;
+
+        position: absolute;
+        top: calc(50% - 40px/2);
+        left: 10%;
+    }
+    #toolbar-logo-name {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 700;
+        font-size: 20px;
+        line-height: 20px;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #FFFFFF;
+    }
+    #item-navigation {
+        width: 255px;
+        height: 40px;
+        display: flex;
+        flex-direction: row;
+        gap: 8px;
+        position: absolute;
+        right: 13%;
+        top: calc(50% - 40px/2);
+    }
+    .navigation-contact {
+        width: 87px;
+    }
+    .navigation-about {
+        width: 69px;
+    }
+    .navigation-contact {
+        width: 83px;
+    }
+    .item-text {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 15px;
+        line-height: 20px;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #FFFFFF;
+    }
+    #mail {
+        display: flex;
+        flex-direction: row;
+        align-items: flex-start;
+        padding: 8px;
+        width: 40px;
+        height: 40px;
+        position: absolute;
+        right: 10%;
+        top: calc(50% - 40px/2);
+    }
+    
+    /* Headline section */
+    #headline {
+        position: relative;
+        height: 739px;
+        width: 39%;
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        left: 10%;
+        top: 72px;
+        padding: 120px 0px;
+    }
+    .my-name  {
+        width: 100%;
+        height: 160px;
+        font-family: 'Crete Round';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 72px;
+        line-height: 82px;
+        color: #F55800;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    .supporting-text{
+        width: 566px;
+        height: 24px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 32px;
+        line-height: 44px;
+        display: flex;
+        align-items: center;
+        color: #FFFFFF;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+    }
+    .description {
+        width: 566px;
+        height: 123px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 20px;
+        line-height: 32px;
+        color: #F4F5F7;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+    }
```

Comment:
At certain points of the desktop version, the headline, description, and the background image get cramped together making it difficult to see the text, as shown below :point_down:  hence it does become responsive. Kindly check and fix this

### Your design

![Screenshot from 2023-01-27 17-56-42](https://user-images.githubusercontent.com/65233860/215162396-ead93140-0bc5-4d71-a578-11f0c9384005.png)


### Figma design

![Screenshot from 2023-01-27 18-13-26](https://user-images.githubusercontent.com/65233860/215163093-00a3da96-3e26-4fdb-839d-2a21e42a4793.png)